prov/gni/src/gnix_msg.c

```diff
@@ -1577,7 +1577,7 @@ ssize_t _gnix_send(struct gnix_fid_ep *ep, uint64_t loc_addr, size_t len,
 	}
 
 	if ((flags & FI_INJECT) && (len > GNIX_INJECT_SIZE)) {
-		GNIX_INFO(FI_LOG_EP_DATA,
+		GNIX_DEBUG(FI_LOG_EP_DATA,
```

Comment:
I agree.  Thanks for catching this.

On Thu, May 5, 2016 at 4:40 PM, Sung-Eun Choi <notifications@github.com>
wrote:

> In prov/gni/src/gnix_msg.c
> <https://github.com/ofi-cray/libfabric-cray/pull/810#discussion_r62266969>
> :
>
> > @@ -1577,7 +1577,7 @@ ssize_t _gnix_send(struct gnix_fid_ep *ep, uint64_t loc_addr, size_t len,
> >  	}
> >
> >  	if ((flags & FI_INJECT) && (len > GNIX_INJECT_SIZE)) {
> > -		GNIX_INFO(FI_LOG_EP_DATA,
> > +		GNIX_DEBUG(FI_LOG_EP_DATA,
>
> I think this should remain GNIX_INFO, because it's the type of message
> that could help an end user.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ofi-cray/libfabric-cray/pull/810/files/522c843df3e3e1730e32521544af8fae7abf35b9#r62266969>
>